tests/foreman/api/test_ldapauthsource.py

```diff
@@ -76,14 +77,14 @@ def test_positive_endtoend_withad(self):
                     location=[self.loc],
                     organization=[self.org],
                 ).create()
-                self.assertEqual(authsource.name, server_name)
+                assert authsource.name == server_name
```

Comment:
@akhil-jha should we parameterize `test_positive_endtoend_withad`? That's also part of pytest conversion effort.

Reply:
Agreed with Jameer. Porting to `assert`, porting away from `subTest` and porting setups to fixtures are core of pytest conversion effort.